cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -240,3 +241,21 @@ def test_import_report(appliance):
     response, = appliance.rest_api.collections.reports.action.execute_action("import", data)
     assert_response(appliance)
     assert response['message'] == 'Skipping Report (already in DB): [{}]'.format(menu_name)
+
+
+@pytest.mark.sauce
+@pytest.mark.tier(3)
+@test_requirements.report
+def test_reports_delete_saved_report(custom_report_values, appliance):
+    """This test case deletes reports from the Saved Reports list.
+    """
+    report = appliance.collections.reports.create(**custom_report_values)
+    report.queue(wait_for_finish=True)
+    view = navigate_to(appliance.collections.saved_reports, 'All')
+    # iterates through every row and checks if the 'Name' column matches the given value
+    for row in view.table.rows():
```

Comment:
Required: As SavedReport is a collection you can instantiate an entity and use simple saved_report.delete() as we have such method for it, and don't iterate all list
saved_report = appliance.collections.saved_reports.instantiate(params)

Reply:
Using `saved_report.delete()` would go the the `Details` page of that saved report and delete it from there. But, I wrote this test case to see if we can select reports from the Saved Reports list and delete them.